### DISCUSSION TITLE
Label backgound color.

### DISCUSSION THREAD

**Discussion by CoutinhoElias** [israel-dryer/ttkbootstrap#520] [General]
Is there a way for a Label to take on the color of a Frame?
The font will be white and the Label background will be the same color as the Frame.

**Comment by ghost** [comment]
If you are using bootstyles then I have found that if the frame is, for example "primary", then making the label "inverse primary" will give white label on primary background.

**Reply by CoutinhoElias** [reply]
Do you have any examples for me to know?

**Reply by zerootoad** [reply]
bootstyle=(INVERSE, PRIMARY)
